src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/JJDisputeMapper.java

```diff
@@ -54,6 +54,7 @@ public abstract class JJDisputeMapper extends BaseMapper {
 	@Mapping(source = "lawyerGiven3Nm", target = "lawyerGivenName3")
 	@Mapping(source = "lawyerSurnameNm", target = "lawyerSurname")
 	@Mapping(source = "noticeOfDisputeGuid", target = "noticeOfDisputeGuid")
+	@Mapping(source = "noticeOfHearingYn", target = "noticeOfHearingYn")
```

Comment:
As far as I know this is not needed as source and target have the same name. You may remove it if you like.

Reply:
Yes, I think you're right, its not needed if the names are the same. I just had it in there to be explicit so we have a complete enumerated list of fields that are mapped (there are several that are unmapped yet).

Reply:
Oh okay, that makes sense.